### DISCUSSION TITLE
Cloudflare DNS Updater

### DISCUSSION THREAD

**Discussion by JesseWebDotCom** [community-scripts/ProxmoxVE#999] [Request script]
### Application Name

Cloudflare DNS Updater

### Website

https://www.cloudflare.com

### Description

For those of us that use Cloudflare, have dynamic IP's, and need an automated way to update our zone when our WAN ip changes - I hacked together the attached script.  Perhaps you'd want to add it so others can use.
[cloudflareddns.sh.md](https://github.com/user-attachments/files/18233885/cloudflareddns.sh.md)

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.
- [X] This is not a game-related request.

**Comment by HearthCore** [comment]
In the meantime, Docker solution:
- https://github.com/favonia/cloudflare-ddns

**Reply by JesseWebDotCom** [reply]
Yep (that's probably the one I used previously) - just trying to do the same now with an lxc

**Comment by knario** [comment]
Hello, thank you very much for the script. 
I currently have it configured in docker (casaOS).
I wanted to ask you if it supports two domains in the same configuration file. for example... mydomain.com, mydomain.org, etc…

**Reply by JesseWebDotCom** [reply]
No, I wrote it for one domain.

**Reply by knario** [reply]
Thank you very much, I will try it with two lxc separately, each one for a domain to see if they do not consume too much.

**Comment by JesseWebDotCom** [comment]
You could ask chat gpt to update ithebscript support multiple domains.  I could do that after the holidays as wellSent from my iPhoneOn Dec 25, 2024, at 9:49 AM, knario ***@***.***> wrote:﻿
Thank you very much, I will try it with two lxc separately, each one for a domain to see if they do not consume too much.

**Comment by Bensonheimer992** [comment]
https://community-scripts.github.io/ProxmoxVE/scripts?id=cloudflare-ddns
@JesseWebDotCom
